handlers/handlers.go

```diff
@@ -24,6 +24,44 @@ func (d *CatalystAPIHandlersCollection) Ok() httprouter.Handle {
 	}
 }
 
+func (d *CatalystAPIHandlersCollection) TranscodeSegment() httprouter.Handle {
+	schemaLoader := gojsonschema.NewStringLoader(TranscodeSegmentRequestSchemaDefinition)
+
+	schema, err := gojsonschema.NewSchema(schemaLoader)
+	if err != nil {
+		panic(err)
```

Comment:
Let's handle it a bit more gracefully

Reply:
I could move the error on program start and panic early.. let me see how

Reply:
Fixed please check this [commit](https://github.com/livepeer/catalyst-api/pull/17/commits/c7d75babcfc286b86ce85ac000c1f18e0fead78a)